homeassistant/components/smappee/sensor.py

```diff
@@ -31,31 +34,31 @@
         "mdi:power-plug",
         ENERGY_WATT_HOUR,
         "power_today",
-        None,
+        ENERGY_WATT_HOUR,
```

Comment:
This is not a device class ?

Reply:
I'm assuming it's device class because that's what is passed in on line 106.

Reply:
nm, that's a different list.

Reply:
It indeed is a device class (will change it to DEVICE_CLASS_ENERGY)